core/src/main/scala/org/apache/spark/sql/execution/columnar/ExternalStoreUtils.scala

```diff
@@ -155,7 +155,7 @@ object ExternalStoreUtils {
         // Already connected to SnappyData in embedded mode.
         Constant.DEFAULT_EMBEDDED_URL + ";host-data=false;mcast-port=0"
       case SplitClusterMode(_, _) =>
-        ServiceUtils.getLocatorJDBCURL(sc) + "/route-query=false"
+        ServiceUtils.getLocatorJDBCURL(sc) + ";route-query=false"
```

Comment:
Why was this not showing error earlier. Any idea? The accepts url was returning ok for the first part of the url but the property route-query=false was not being set as intended. What problems surfaced due to this.